### PR TITLE
Update Nexus on fire and destroyed

### PR DESCRIPTION
**Pull request by deweiZou**

### CODE REVIEW

File: src/main/java/lol/ui/BattlefieldView.java [hunk 1]
```diff
@@ -65,7 +65,32 @@ ImageView championView(Champion champion) {
 
   ImageView nexusView(Nexus nexus) {
     int teamID = nexus.teamOfNexus();
-    return(sprites.nexusesView(teamID));
+    double hpPercentage = (nexus.currentHP())/(double)(nexus.initialHP());
+    switch(nexus.teamOfNexus()) {
```
Comment (ptal) [inline src/main/java/lol/ui/BattlefieldView.java:69]: The code in the Nexus.BLUE and Nexus.RED branches are the same. Try to find a way to group them together.
A way is to have an array of nexus, e.g.,
```java
private Image[] nexusOnFire;
private Image[] nexusDestroyed;
```

Then, `nexusDestroyed[Nexus.BLUE]` to access the blue / red images.

**Review by ptal — CHANGES_REQUESTED** [review]
Cool stuff. Did you make the images yourself or is it subjected to copyright?

**Comment by deweiZou** [comment]
Yes, the image is totally create by myself, actually it's based on given image, only thing I did just draw something on it.
________________________________
Von: Pierre Talbot ***@***.***>
Gesendet: Donnerstag, 20. Mai 2021 08:31
An: ptal/lol2D ***@***.***>
Cc: ZOU Dewei ***@***.***>; Author ***@***.***>
Betreff: Re: [ptal/lol2D] Update Nexus on fire and destroyed (#53)


@ptal requested changes on this pull request.

Cool stuff. Did you make the images yourself or is it subjected to copyright?

________________________________

In src/main/java/lol/ui/BattlefieldView.java<https://github.com/ptal/lol2D/pull/53#discussion_r635801865>:

> @@ -65,7 +65,32 @@ ImageView championView(Champion champion) {

   ImageView nexusView(Nexus nexus) {
     int teamID = nexus.teamOfNexus();
-    return(sprites.nexusesView(teamID));
+    double hpPercentage = (nexus.currentHP())/(double)(nexus.initialHP());
+    switch(nexus.teamOfNexus()) {


The code in the Nexus.BLUE and Nexus.RED branches are the same. Try to find a way to group them together.
A way is to have an array of nexus, e.g.,

private Image[] nexusOnFire;
private Image[] nexusDestroyed;

Then, nexusDestroyed[Nexus.BLUE] to access the blue / red images.

**Comment by deweiZou** [comment]
And ill change it
________________________________
Von: Pierre Talbot ***@***.***>
Gesendet: Donnerstag, 20. Mai 2021 08:31
An: ptal/lol2D ***@***.***>
Cc: ZOU Dewei ***@***.***>; Author ***@***.***>
Betreff: Re: [ptal/lol2D] Update Nexus on fire and destroyed (#53)


@ptal requested changes on this pull request.

Cool stuff. Did you make the images yourself or is it subjected to copyright?

________________________________

In src/main/java/lol/ui/BattlefieldView.java<https://github.com/ptal/lol2D/pull/53#discussion_r635801865>:

> @@ -65,7 +65,32 @@ ImageView championView(Champion champion) {

   ImageView nexusView(Nexus nexus) {
     int teamID = nexus.teamOfNexus();
-    return(sprites.nexusesView(teamID));
+    double hpPercentage = (nexus.currentHP())/(double)(nexus.initialHP());
+    switch(nexus.teamOfNexus()) {


The code in the Nexus.BLUE and Nexus.RED branches are the same. Try to find a way to group them together.
A way is to have an array of nexus, e.g.,

private Image[] nexusOnFire;
private Image[] nexusDestroyed;

Then, nexusDestroyed[Nexus.BLUE] to access the blue / red images.

**Comment by deweiZou** [comment]
changed the code

**Pull request merged**